package.json

```diff
@@ -66,7 +66,7 @@
     "lint-staged": "^10.2.10",
     "minimist": "^1.2.0",
     "npm-run-all": "^4.1.5",
-    "prettier": "~1.14.0",
+    "prettier": "^2.3.1",
```

Comment:
![image](https://user-images.githubusercontent.com/31695475/121777211-81b33a00-cbc3-11eb-905f-b294511c853c.png)
TS template literal types that cannot be handled.